posts/2022-09-01-22.0.0.10-beta.adoc

```diff
@@ -0,0 +1,581 @@
+// important becuase it is supposed to grab the readers attention.  Make sure to keep the blank lines 
+//
+// Throughout the doc, replace 22.0.0.10-beta with the version number of Open Liberty, eg: 22.0.0.2-beta
+// // // // // // // //
+
+The link:{url-about}[Open Liberty] 22.0.0.10-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* <<java, Java 19 Support>>
+* <<xml, XML Binding 4.0>>
+* <<bell, BELL SPI Visibility and BELL Properties>>
+* <<microprofile, MicroProfile Telemetry 1.0>>
+* <<security, Distributed Security Caches>>
+
+// // // // // // // //
+// In the preceding section:
+// Change SUB_FEATURE_TITLE to the feature that is included in this release and
+// change the SUB_TAG_1/2/3 to the heading tags
+//
+// However if there's only 1 new feature, delete the previous section and change it to the following sentence:
+// "The link:{url-about}[Open Liberty] 22.0.0.10-beta includes SUB_FEATURE_TITLE"
+// // // // // // // //
+
+// https://github.com/OpenLiberty/open-liberty/issues/22166
+[#java]
+== Java 19 Support
+Java 19 is coming soon, and with it the following features and changes:
+
+* 405: Record Patterns (Preview)
+* 422: Linux/RISC-V Port
+* 424: Foreign Functions & Memory API (Preview)
+* 425: Virtual Threads (Preview)
+* 426: Vector API (Fourth Incubator)
+* 427: Pattern Matching for switch (Third Preview)
+* 428: Structured Concurrency (Incubator)
+
+By taking advantage of trying out the new changes in Java 19 now, you will get more time to review your applications, microservices and runtime environments which means you will be a step ahead when it becomes generally available.
+
+To give this a try, link:https://jdk.java.net/19/[download the early access release of Java 19], download and install the link:https://openliberty.io/downloads/#runtime_betas[22.0.0.10-beta] version of Open Liberty, edit your Liberty server's link:https://openliberty.io/docs/22.0.0.10/reference/config/server-configuration-overview.html#server-env[server.env] file to point JAVA_HOME to your Java 19 installation and start testing!
+
+For more information on Java 19, visit the Java 19 link:https://jdk.java.net/19/release-notes[release notes page], link:https://download.java.net/java/early_access/jdk19/docs/api/[API Javadoc page] or link:https://openliberty.io/docs[download page].
+For more information on Open Liberty, please visit our documentation page.
+
+As work is made towards full Java 19 support, there may functionality that might not be 100% ready yet but is in progress to be complete soon!
+
+// https://github.com/OpenLiberty/open-liberty/issues/22092
+[#xml]
+== XML Binding 4.0
+Jakarta XML Binding 4.0 is an API for automating the mapping between XML documents and Java objects. Support provides the updated XML Binding 4.0 API, it's Reference Implementation, and introduces the xmlBinding-4.0 feature to Open Liberty.

```

Comment:
"for automating the mapping between XML documents and Java objects when writing XML documents in Java" or something like that? Can we give a brief sentence about when you'd use this?

Reply:
@neuwerk could you at least include something about what changed in 4.0 compared to 3.0?  I think that would help for sure and I think Laura would like an example of how it is used or what you can do now that you couldn't do before.

Reply:
no changes